source/retryable-writes/tests/unified/handshakeError.yml

```diff
@@ -0,0 +1,77 @@
+description: "retryable writes handshake failures"
+
+schemaVersion: "1.3"
+
+runOnRequirements:
+  - minServerVersion: "4.2"
+    topologies: [replicaset, sharded, load-balanced]
+    auth: true
+
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      observeEvents: [commandStartedEvent, connectionCheckOutStartedEvent]
+      uriOptions: { retryReads: true, retryWrites: true, maxPoolSize: 5 }
```

Comment:
Similar comment as I left in the retryable reads test about the default valuse and `maxPoolSize`.